integrationtests/tests/fixtures/mod.rs

```diff
@@ -127,6 +166,11 @@ where
         fixtures.lightning,
     )
     .await;
+
+    if let Ok(backup_dir) = env::var("FM_TEST_DB_BACKUP_DIR") {
```

Comment:
I generally don't like the part of taking a snapshot from integration tests etc. I don't think we are going to get good coverage from it.

Developers of every module needs to prepare their own snapshot to test from on their own, seeding with intelligently crafted data, IMO.

Reply:
Removed